CONTRIBUTING.md

```diff
@@ -30,8 +30,9 @@ To get started, please [sign](https://cla-assistant.io/tldr-pages/tldr) the
 The basic format of a `tldr` page is a set of concrete usage examples.
 Here are a few guidelines to get started:
 
-1. Try to keep pages at around 5 examples. Pages can be longer if needed, but don't exceed 8 examples.
-   Remember, it's OK if the page doesn't cover everything; that's what `man` is for.
+1. Try to keep pages at around 5 examples. Pages can be longer if needed or shorter where that makes more sense, but 
+   specify one example at least and 8 examples at most.
+   Remember it's OK if the page doesn't cover everything; that's what `man` is for.
```

Comment:
Why remove the comma? I think it reads better with it.

Reply:
I probably accidentally pressed `d` in my editor or so… I wasn't intending to remove it. I'll re-add it.